tests/test_default_project.py

```diff
@@ -0,0 +1,37 @@
+from typing import Any, Dict, Text
+
+from warnings import WarningMessage
+from _pytest.pytester import Testdir
+import pytest
+
+from rasa.__main__ import create_argument_parser
+import rasa.cli.data
+import rasa.cli.scaffold
+import rasa.cli.train
+import rasa.shared.utils.io
+
+
+def _warning_should_be_filtered_out(warning: WarningMessage) -> bool:
+    # we filter out `gelu` warnings because of this issue:
```

Comment:
This is more for my own knowledge - what are `gelu` warnings?

Reply:
`gelu` is an [activation function](https://en.wikipedia.org/wiki/Activation_function) for neurons in a neural network. There are many different action functions as people keep on coming up with new ones which supposedly help the networks to learn better / quicker etc.

Reply:
the warning is just a tensorflow thing because they've moved the implementation of that function to a different package